active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,512 @@
+{
+    if (args.Reason == InfoBarCloseReason.CloseButton) {
+        // if scenario failed
+        args.Cancel = true;
+    }
+    
+}
+```
+
+# Inputs and Accessibility
+
+## UI Automation Patterns 
+
+InfoBar will use a Pane control pattern for inline notifications and will implement a custom "information" Landmark.
+
+### Keyboard Navigation 
+- No action is needed to invoke the InfoBar
+- After navigating to the control via tabbing, the user is automatically focused into the control once navigating to the first actionable item.
+- Once focused, tabbing will iterate through all actionable items in the control in order. When tab is pressed on the last element in the InfoBar, focus will continue throughout the page.
+  - A user can also use the left and right arrow keys to navigate between the available buttons
+- To close the InfoBar the action or close button needs to be pressed.
+  - Note: Escape will not close the InfoBar and will instead bubble up the command to the parent components.
+
+### Gamepad
+
+- The actionable items in the control can be navigated to via spatial navigation.
+- The 'A' button will interact with the item in focus, such as "press" the action or close buttons
+- The InfoBar can be closed via pressing the 'X' close button, pressing the action button, or the 'B' button which will return focus to the element previously in focus.
+### Assistive Technologies
+
+InfoBar will leverage the existing APIs used by Windows Notifications. 
+
+The behavior of the InfoBar will change for assistive technologies like Narrator depending on the Severity set by the developer. As Error and Warning InfoBars are intended to be used for scenarios that directly impact app experience they should interrupt the user more than InfoBars that are informational. View [NotificationProcessing ](https://docs.microsoft.com/en-us/windows/win32/api/uiautomationcore/ne-uiautomationcore-notificationprocessing) docs for more information on the varied intended behavior.
+
+| Severity |  NotificationProcessing | Behavior in Narrator|
+|:--- | :---| :---|
+| Error | NotificationProcessing_ImportantAll| Add new item to the end of the queue. It doesn’t matter the source, what is currently speaking or any other items in the queue queued with a higher priority than normal text. <br><br> This will cause it to get spoken after the current utterance/string finishes. Meaning it won't interrupt the speech but it will queue itself next to be spoken, ahead of anything already queued. <br><br> Focus and keyboard will NOT interrupt or flush, pressing control will silence and flush all |
+| Warning | NotificationProcessing_ImportantAll| See above |
+| Success | Processing_All| Add new item to the end of the queue. <br><br> Will be added to the end of the queue meaning all existing queued text will need to be spoken before this text will be spoken. <br><br>Focus, keyboard and control will silence/flush them all
+| Default | Processing_All | See above
+
+#### Narrator
```

Comment:
Why are we talking about Narrator? Are we making updates to that app, or does this section apply to any screen reader?

Reply:
We are not making updates to Narrator however when describing the effect of the [UIA Event](https://docs.microsoft.com/en-us/windows/win32/winauto/uiauto-eventsoverview) to screen readers I've seen Narrator used in other specs as example behavior.